newIDE/app/src/MainFrame/EditorContainers/HomePage/ManageSection/index.js

```diff
@@ -117,7 +119,7 @@ const ManageSection = ({
               <GameDetails
                 game={openedGame}
                 project={project}
-                onGameUpdated={onRefreshGames}
+                onGameUpdated={onGameUpdated}
```

Comment:
This is part of the fix. When the game is updated, don't refresh the list (that will create a "blink") but just ask the parent to do its job of updating the game.